controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -212,12 +213,15 @@ func (r *BareMetalHostReconciler) Reconcile(ctx context.Context, request ctrl.Re
 	}
 
 	ready, err := prov.TryInit()
-	if err != nil {
-		return ctrl.Result{}, errors.Wrap(err, "failed to check services availability")
-	}
-	if !ready {
+	if err != nil || !ready {
+		var msg string
+		if err == nil {
+			msg = NotReady
+		} else {
+			msg = err.Error()
+		}
 		provisionerNotReady.Inc()
-		reqLogger.Info("provisioner is not ready", "RequeueAfter:", provisionerNotReadyRetryDelay)
+		reqLogger.Info("provisioner is not ready", "Error", msg, "RequeueAfter:", provisionerNotReadyRetryDelay)
```

Comment:
"Error:"

Reply:
fixed

Reply:
This was wrong btw, please don't add colons to structured logging